README.md

```diff
@@ -32,37 +32,24 @@ export GITHUB_LABELS_TOKEN="thisismytoken"
 Load a new shell or source your rc file to take into account this new
 environment variable (_e.g._ `source $HOME/.zshrc`).
 
-#### Install the `labels` tool
-
-Make sure [Yarn](https://yarnpkg.com) is installed on your machine, then, at the
-root of this repository, install `labels`:
+#### Build the `fun-config` Docker image
 
 ```bash
-$ cd path/to/fun-config
-$ yarn install
+$ make build
 ```
 
 ### Create default labels for a repository
 
-Once `labels` is installed, you can use our default labels configuration to
-initialize labels from a repository:
+Once the `fun-config` image has been successfully build, you can use our default
```

Comment:
s/build/built

Reply:
Every time. Always. Damned. One day I won't make this mistake.